conf/application.conf

```diff
@@ -222,13 +221,16 @@ guidance {
   aiCodesForContainers = "https://www.gov.uk/government/publications/appendix-1-de-110-requested-and-previous-procedure-codes/requested-procedure-10-permanent-export-or-dispatch#additional-information-de-22-1"
   appointSomeoneToDealWithCustomsOnYourBehalf = "https://www.gov.uk/guidance/appoint-someone-to-deal-with-customs-on-your-behalf"
   cdsDeclarationSoftware = "https://www.gov.uk/guidance/list-of-software-developers-providing-customs-declaration-support"
+  cdsRegister = ${urls.cdsRegister}
   cdsTariffCompletionGuide = "https://www.gov.uk/government/publications/uk-trade-tariff-cds-volume-3-export-declaration-completion-guide"
   clearingGoodsFromToUK = "https://www.gov.uk/guidance/national-clearance-hub-for-goods-entering-leaving-or-transiting-the-eu"
   commodityCode0306310010 = "https://www.trade-tariff.service.gov.uk/commodities/0306310010#export"
   commodityCode2208303000 = "https://www.trade-tariff.service.gov.uk/commodities/2208303000#import"
   commodityCode9306909000 = "https://www.trade-tariff.service.gov.uk/xi/commodities/9306909000?country=KP#export"
   commodityCodes = ${urls.commodityCodes}
+  eoriService = ${urls.eoriService}
```

Comment:
I might be missing some context, but why is this "alias" required? Could not the `urls.eoriService` be used?